plugin.xml

```diff
@@ -8,6 +8,9 @@
   <repo>git+https://github.com/emilbayes/cordova-plugin-android-downloadmanager.git</repo>
   <issue>https://github.com/emilbayes/cordova-plugin-android-downloadmanager/issues</issue>
   <platform name="android">
+    <js-module src="www/index.js" name="DownloadManager">
```

Comment:
I'm happy to add this if it makes it easier to use

Reply:
It changes the plugin behaviour, it's loaded with cordova runtime this way.